src/figuresClasses.ts

```diff
@@ -1,19 +1,97 @@
-export interface Figure {
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum Colors {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
 
+export interface Figure {
+  shape: Shapes,
+  color: Colors,
+  getArea(): number,
 }
 
-export class Triangle {
+function validateSides(...sides: number[]): void {
+  if (sides.some((side) => side <= 0)) {
+    throw new Error('Each side value should be bigger than 0!');
+  }
+}
 
+function roundValue(area: number): number {
+  return Math.floor(area * 100) / 100;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape = Shapes.Triangle;
```

Comment:
```suggestion
  readonly shape = Shapes.Triangle;
```
I think shape can be protected from changes.